extensions/expression-language/Expression.cpp

```diff
@@ -632,7 +632,9 @@ Value expr_toDate(const std::vector<Value> &args) {
 #endif  // EXPRESSION_LANGUAGE_USE_DATE
 
 Value expr_now(const std::vector<Value> &args) {
-  return Value(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
+  using namespace std::chrono;
+  int64_t unix_time_ms = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
```

Comment:
Please consider using direct-list-initialization to avoid narrowing implicit conversions.
```suggestion
  int64_t unix_time_ms{duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count()};
```